datasets/api.py

```diff
@@ -247,6 +247,51 @@ def get_queryset(self):
         ).select_related('datapoint', 'created_by', 'updated_by', 'resolved_by')
```

Comment:
Remember to check all reference to fields like `updated_by` that were changed when switching to the new modification_tracking code.